ts/Dashboard/SerializeHelper/HTMLTableConverterHelper.ts

```diff
@@ -60,7 +59,10 @@ function jsonSupportFor(obj: unknown): obj is HTMLTableConverter {
  * @return {HTMLTableConverter} New Data Converter object created from serialized object
  */
 function fromJSON(json: HTMLTableConverterHelper.JSON): HTMLTableConverter {
-    return new HTMLTableConverter(json.options);
+    const id = json.tableElementId,
+        tableElement = id ? document.getElementById(id) : null;
+
+    return new HTMLTableConverter(json.options, tableElement);
```

Comment:
What in the case when ID is null, does it return the HTMLTableConverter?

Reply:
Yes, it does, it creates the instance of the `HTMLTableConverter`, but without the tableElement.